packages/twenty-front/src/modules/object-metadata/hooks/useCreateOneObjectMetadataItem.ts

```diff
@@ -42,15 +35,7 @@ export const useCreateOneObjectMetadataItem = () => {
     return createdObjectMetadata;
   };
 
-  const findManyRecordsCache = async () => {
```

Comment:
this is not needed as the PrefetchQueries will update themselves as ObjectMetadataItems are refreshed